README.md

```diff
@@ -213,11 +213,14 @@ sublime-phpcs is insanely powerful, but if you'd prefer automatic linting, [Subl
 - Install [linter-phpcs](https://atom.io/packages/linter-phpcs) via Atom's package manager.
 - Run `which phpcs` to get your `phpcs` executable path.
 - Enter your `phpcs` executable path and one of the coding standards specified above (e.g. `WordPress`, `WordPress-VIP`, etc.).
+- Change the Tab Width setting to `4`.
```

Comment:
Do Atom users know where to find that setting ? Or should the instruction on where to change this be made more specific ?